tools/build.cmd

```diff
@@ -170,5 +170,5 @@ REM
 REM ##########################################################################################
 :build_config_ninja
 cmake -G "Ninja" -DCMAKE_MAKE_PROGRAM="!NINJA!" -DCMAKE_TOOLCHAIN_FILE="!VCPKG_CMAKE!" !CONFIG! "!ROOT!"
-%NINJA%
+"%NINJA%"
```

Comment:
Would this work if the full path of ninja has already been quoted?

Reply:
> Would this work if the full path of ninja has already been quoted?

No, it won't work. The action (setting `%NINJA%` to something that contains double quotes) here is already sounding like a bug to me.